source/includes/_protocol.md

```diff
@@ -13,28 +13,29 @@ API and authorize access to our user's accounts.
 ## Endpoints
 
 The API is now versioned, in order to support backwards-incompatible changes.
-The current version is **v1**. This means the base URLs are:
-
-- Testing Sandbox: <https://api-sandbox.gresb.com/api/v1>
-- Production: <https://api.gresb.com/api/v1>
-
+With the introduction of portfolio-level endpoints, the current application uses both the **v0** and **v1**. This means the base URLs are:
 
+- Testing Sandbox: <https://api-sandbox.gresb.com/api/v0> for portfolio-level, and <https://api-sandbox.gresb.com/api/v1> for asset-level APIs.
+- Production: <https://api.gresb.com/api/v0> for portfolio-level, and <https://api.gresb.com/api/v1> for asset-level APIs.
 
 The following paths/endpoints are available:
 
-| Path                                                            | HTTP verbs/methods |
-|-----------------------------------------------------------------|--------------------|
-| [/certifications](#certifications)                              | GET                |
-| [/user](#users)                                                 | GET                |
-| [/entities](#reporting-entities)                                | GET                |
-| [/entities/{entity_id}](#reporting-entities)                    | GET                |
-| [/entities/{entity_id}/assets](#asset-data)                     | GET, POST          |
-| [/entities/{entity_id}/assets/{gresb_asset_id}](#asset-data)    | GET, PATCH, DELETE |
-| [/entities/{entity_id}/assets/batches](#batch-asset-operations) | POST               |
-
-<aside class="notice">
-  <strong>NOTE:</strong> Since the API changes for the 2020 survey, we do not provide backward compatibility, thus <strong>all URLs with version 0 are no longer supported</strong> and will return an error message.
-</aside>
+| Path                                                                                | HTTP verbs/methods | Version
```

Comment:
nice overview 👌 should be somewhere in a section at the top of the document. Which it is I believe.